server/JsDbg.Gdb/GdbDebugger.cs

```diff
@@ -36,6 +35,27 @@ class GdbDebugger : IDebugger {
             }
         }
 
+        void NotifyDebuggerChange(DebuggerChangeEventArgs.DebuggerStatus status) {
+            this.DebuggerChange?.Invoke(this, new DebuggerChangeEventArgs(status));
+        }
+
+        void HandleGdbEvent(object sender, string ev) {
+            if (ev[0] != '%')
+                return;
+
+            DebuggerChangeEventArgs.DebuggerStatus oldStatus = debuggerStatus;
+
+            if (ev == "%cont")
+                debuggerStatus = DebuggerChangeEventArgs.DebuggerStatus.Waiting;
+            else if (ev == "%stop")
+                debuggerStatus = DebuggerChangeEventArgs.DebuggerStatus.Break;
+            else if (ev == "%exit")
+                debuggerStatus = DebuggerChangeEventArgs.DebuggerStatus.Detaching;
+

```

Comment:
What about process and thread changes?

Reply:
You'll have to merge https://github.com/MicrosoftEdge/JsDbg/pull/46 before I can work on those :) The current jsdbg_gdb_support branch is too old for those events.

(But I'm happy to wait with merging this until after that pull request, and then I can add those too)

Reply:
Ah yes, of course. :)

Approved that merge. Can you just leave a TODO for those here?

Reply:
TODO added (in the python file)